### PR TITLE
Refactor simorgh-e2e notifications into alarms channels

### PR DESCRIPTION
**Pull request by DemonusPC**
**Overall change:**
Added the more slack messages

**Code changes:**

- More slack messages
- Made channels a passable variable

---

- [ ] I have assigned myself to this PR and the corresponding issues
- [ ] Automated (jest and/or cypress) tests added (for new features) or updated (for existing features)
- [ ] This PR requires manual testing

### CODE REVIEW

File: Jenkinsfile-e2e [hunk 1]
```diff
@@ -53,7 +53,8 @@ node {
               ]
 
               if (env.BRANCH_NAME == 'latest') {
-                notifySlack(messageParameters)
+                notifySlack(messageParameters, '#si_repo-simorgh')
```
Comment (chris-hinds) [inline Jenkinsfile-e2e]: Could these slack channel names be moved to a var at the top of the file and only referenced here

**Review by chris-hinds — CHANGES_REQUESTED** [review]

File: Jenkinsfile-e2e [hunk 2]
```diff
@@ -102,7 +104,7 @@ node {
                   colour: 'good',
                   emoji: '🤟'
                 ]
-                notifySlack(messageParameters)
+                // notifySlack(messageParameters, '#si_repo-simorgh')
```
Comment (jamesbhobbs) [inline Jenkinsfile-e2e]: why is this commented out?

File: Jenkinsfile-e2e (hunk 1 above)
Comment (chris-hinds) [reply Jenkinsfile-e2e]: will make it much easier to update channel names if we ever need to.

File: Jenkinsfile-e2e [hunk 3]
```diff
@@ -112,7 +114,7 @@ node {
                   colour: 'danger',
                   emoji: '😱',
                 ]
-                notifySlack(messageParameters)
+                notifySlack(messageParameters, '#si_repo-simorgh')
```
Comment (jamesbhobbs) [inline Jenkinsfile-e2e]: I don't follow the logic here, why are these only reporting to repo simorgh?

File: Jenkinsfile-e2e-test [hunk 4]
```diff
@@ -53,7 +53,8 @@ node {
               ]
 
               if (env.BRANCH_NAME == 'latest') {
-                notifySlack(messageParameters)
+                notifySlack(messageParameters, '#si_repo-simorgh')
+                notifySlack(messageParameters, '#simorgh-alarms-live')
```
Comment (jamesbhobbs) [inline Jenkinsfile-e2e-test]: these should be test

File: Jenkinsfile-e2e (hunk 3 above)
Comment (jamesbhobbs) [reply Jenkinsfile-e2e]: Each of these should also report to #simorgh-alarms-live I think

File: Jenkinsfile-e2e (hunk 1 above)
Comment (DemonusPC) [reply Jenkinsfile-e2e]: yes will look into that

File: Jenkinsfile-e2e (hunk 2 above)
Comment (DemonusPC) [reply Jenkinsfile-e2e]: > Continue posting in the si_repo-simorgh channel only if any pipeline step fails.
It says it on the issue. I haven't finished yet tho

Comment (chris-hinds) [reply Jenkinsfile-e2e]: we should probably treat. unstable and aborted builds as failures too, so will be good to send to the alarms channel

**Comment by chris-hinds** [comment]
@DemonusPC where we specify `env: 'ALL',` in the slack message params can this be updated to reflect the new pipeline please.

I.e. the Jenkins-e2e pipeline should say `env: Local and Production`
the Jenkins-e2e-test pipeline should say `env: Test`

**Comment by chris-hinds** [comment]
In the post steps can all the build stages except `SUCCESS` send to messages to both slack channels please

File: Jenkinsfile-e2e [hunk 5]
```diff
@@ -97,43 +105,47 @@ node {
...
                   buildStatus: 'SUCCESS',
-                  env: 'ALL',
+                  env: 'Local and Production',
                   branch: env.BRANCH_NAME,
                   colour: 'good',
                   emoji: '🤟'
                 ]
-                notifySlack(messageParameters)
+                notifySlack(messageParameters, channels.simorgh)
+                notifySlack(messageParameters, channels.liveAlarms)
```
Comment (chris-hinds) [inline Jenkinsfile-e2e]:
```suggestion
```

**Review by chris-hinds — CHANGES_REQUESTED** [review]

**Review by chris-hinds — APPROVED** [review]

**Review by j-pendlebury — APPROVED** [review]
LGTM 👍

**Pull request merged**